### PR TITLE
Enabled xml doc generation

### PR DESCRIPTION
**Pull request by ajaybhargavb**
@NTaylorMullen

### CODE REVIEW

**Comment by NTaylorMullen** [comment]
:shipit:

File: src/Microsoft.AspNetCore.Localization/IRequestCultureFeature.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ public interface IRequestCultureFeature
         /// <summary>
         /// The <see cref="IRequestCultureProvider"/> that determined the request's culture information.
         /// If the value is <c>null</c> then no provider was used and the request's culture was set to the value of
-        /// <see cref="RequestLocalizationOptions.DefaultRequestCulture"/>.
+        /// <see cref="Builder.RequestLocalizationOptions.DefaultRequestCulture"/>.
```
Comment (hishamco) [inline src/Microsoft.AspNetCore.Localization/IRequestCultureFeature.cs]: @ajaybhargavb why you are adding **Builder**?!! Is the xml doc generation will discover that after building process or there's something else

Comment (pranavkm) [inline src/Microsoft.AspNetCore.Localization/IRequestCultureFeature.cs]: `RequestLocalizationOptions` is located under `Microsoft.AspNetCore.Builder` which isn't referenced here.

Comment (hishamco) [inline src/Microsoft.AspNetCore.Localization/IRequestCultureFeature.cs]: Oops, you know what this the first time I notice that, I thought it's under ```Microsoft.AspNetCore.Localization``` , It's my fault .. thanks

**Pull request merged**